app/views/comments/new.html.erb

```diff
@@ -0,0 +1 @@
+<%= render 'shared/comment-form' %>
```

Comment:
I would put this in the `app/views/comments` folder and name the file `_form.html.erb` rather than using the shared directory.